apps/frontend/src/pages/access-management/libs/enums/name-column-table.enum.ts

```diff
@@ -0,0 +1,10 @@
+import { type ColumnDef } from "@tanstack/react-table";
+
+import { type User } from "../types/user.type.js";
+
+const columns: ColumnDef<User>[] = [
+	{ accessorKey: "name", header: "Name" },
+	{ accessorKey: "createdAt", header: "Created At" },
+];
```

Comment:
I think this fits better as constants rather than enums. If so, then B22 too, but maybe hold off for now